Makefile

```diff
@@ -61,22 +117,20 @@ constraints.txt: dsl/requirements.txt fv3core/requirements.txt pace-util/require
 	sed -i.bak '/\@ git+https/d' constraints.txt
 	rm -f constraints.txt.bak
 
-physics_savepoint_tests:
-	$(MAKE) -C fv3gfs-physics $@
-
-physics_savepoint_tests_mpi:
-	$(MAKE) -C fv3gfs-physics $@
+physics_savepoint_tests: build
+	TARGET=physics $(MAKE) get_test_data
+	$(CONTAINER_CMD) $(CONTAINER_FLAGS) bash -c "pip3 list && cd $(PACE_PATH) && pytest --data_path=$(EXPERIMENT_DATA_RUN)/physics/ $(TEST_ARGS) $(PHYSICS_THRESH_ARGS) $(PACE_PATH)/fv3gfs-physics/tests/savepoint"
 
-update_submodules_venv:
-	if [ ! -f $(CWD)/external/daint_venv/install.sh  ]; then \
-                git submodule update --init external/daint_venv; \
-        fi
+physics_savepoint_tests_mpi: build
+	TARGET=physics $(MAKE) get_test_data
```

Comment:
Do you not have to set `EXPERIMENT` for physics test? It should be using `c12_6ranks_baroclinic_dycore_microphysics`

Reply:
The physics makefile already defaults to that experiment name. EXPERIMENT shouldn't be hard-coded or set at this level, since it should be possible to set the experiment with an environment variable when executing the workflow. Otherwise we could never have more than one experiment for physics tests.

If we remove the physics makefile, this would have to be set by the user when calling `make physics_savepoint_tests`.